ckan/tests/controllers/test_user.py

```diff
@@ -183,10 +185,8 @@ def test_non_root_user_logout_url_redirect(self):
 
         logout_url = url_for(controller='user', action='logout')
         logout_response = app.get(logout_url, status=302)
-        try:
-            final_response = helpers.webtest_maybe_follow(logout_response)
-        except Exception as e:
-            assert_true('/my/prefix/user/logout' in e.message)
+        assert_equal(logout_response.status_int, 302)
```

Comment:
Isn't this already implicitly checked by supplying `status` in the `app.get` above? That will fail if the status is something other then the one given, won't it?